src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -266,10 +276,16 @@ trait FastSync {
       requestedReceipts = requestedReceipts - handler
     }
 
+    private def validateBlocks(requestedHashes: Seq[ByteString], blockBodies: Seq[BlockBody]): Boolean = (requestedHashes zip blockBodies)
```

Comment:
Minor comment up to discussion, should check whether the length of the `requestedHashes` and `blockBodies` is the same?